.env

```diff
@@ -43,12 +43,12 @@ REACT_APP_FORTMATIC_SITE_VERIFICATION="LzjrtdM7hqVJfvvA"
 
 # Domain regex (to detect environment)
 REACT_APP_DOMAIN_REGEX_LOCAL="^(:?localhost:\d{2,5}|(?:127|192)(?:\.[0-9]{1,3}){3})"
-REACT_APP_DOMAIN_REGEX_PR="^pr\d+--cowswap\.review|^(swap-dev-git-[\w\d-]+|swap-\w{9}-)cowswap\.vercel\.app"
-REACT_APP_DOMAIN_REGEX_DEV="^(cowswap\.dev|dev\.swap\.cow\.fi|swap-develop\.vercel\.app)"
-REACT_APP_DOMAIN_REGEX_STAGING="^(cowswap\.staging|staging\.swap\.cow\.fi|swap-staging\.vercel\.app)"
+REACT_APP_DOMAIN_REGEX_PR="^pr\d+--CoW Swap\.review|^(swap-dev-git-[\w\d-]+|swap-\w{9}-)CoW Swap\.vercel\.app"
+REACT_APP_DOMAIN_REGEX_DEV="^(CoW Swap\.dev|dev\.swap\.cow\.fi|swap-develop\.vercel\.app)"
+REACT_APP_DOMAIN_REGEX_STAGING="^(CoW Swap\.staging|staging\.swap\.cow\.fi|swap-staging\.vercel\.app)"
 REACT_APP_DOMAIN_REGEX_PROD="^(swap\.cow\.fi|swap-prod\.vercel\.app)$"
 REACT_APP_DOMAIN_REGEX_BARN="^(barn\.cow\.fi|swap-barn\.vercel\.app)$"
-REACT_APP_DOMAIN_REGEX_ENS="(:?^cowswap\.eth|ipfs)"
+REACT_APP_DOMAIN_REGEX_ENS="(:?^CoW Swap\.eth|ipfs)"
```

Comment:
Please revert this change.
Those are urls patters

Reply:
yep, i must have find and replace by mistake
reverting